app/components/PortableTextComponents.tsx

```diff
@@ -0,0 +1,72 @@
+import { Image } from "~/components/Image"
+import { ExternalLink } from "lucide-react"
+import { Separator } from "~/components/ui/separator"
+
+const PortableTextComponents = {
+  types: {
+    image: ({
+              value,
```

Comment:
prettier, are you drink

Reply:
thats weird, it all works good for me locally. no worries, it will get formatted eventually